### DISCUSSION TITLE
level=error msg="invalid bool value: true'"

### DISCUSSION THREAD

**Discussion by reori** [photoprism/photoprism#3615] [Q&A] [answered]
Hi I am using photoprism on unraid.  
I re-installed the app to use a mariaDB database.
I am confident the database, username, password, etc. are all configured and working correctly.
But I cannot seem to start Photoprism.  Each time Photoprism tries to start, this error appears in the logs:

/opt/photoprism/bin/photoprism start
time="2023-08-07T21:46:50+02:00" level=error msg="invalid bool value: true'"

Can anyone help me understand how to fix this? Thanks in advance!

**Comment by graciousgrey** [comment, marked as answer]
This could be a problem with your database. Turning on the [debug logs](https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3) might provide further insight.

You can find more information on how to troubleshoot database issues here:
- https://docs.photoprism.app/getting-started/troubleshooting/sqlite/#migrating-to-mariadb
- https://docs.photoprism.app/getting-started/troubleshooting/mariadb/

I would recommend to check your database schema first:
https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#incompatible-schema

**Reply by reori** [reply]
Thank you.

I switched on the debugging and the logs show this:

/opt/photoprism/bin/photoprism start
time="2023-08-08T10:17:26+02:00" level=debug msg="settings: loaded from /photoprism/storage/config/settings.yml"
time="2023-08-08T10:17:31+02:00" level=debug msg="config: running on 'Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz', 17 GB memory detected"
time="2023-08-08T10:18:31+02:00" level=error msg="invalid bool value: true'"

I think the database settings are fine and I had mariadb working with Photoprism previously.  But then I reconfigured my unraid and changed some cache settings.  In doing so I deleted the Photoprism app data. Then I reinstalled photoprism and it works with the default sqlite.  When I changed to a fresh mysql database, this error arose.

**Reply by reori** [reply]
Ok I figured it out.  There was a syntax error in the Photoprism settings relating to the database login details.  
My own noob misunderstanding.

Thanks again for your kind help!
